codersdk/deployment.go

```diff
@@ -530,3 +545,35 @@ func (c *Client) AppHost(ctx context.Context) (AppHostResponse, error) {
 	var host AppHostResponse
 	return host, json.NewDecoder(res.Body).Decode(&host)
 }
+
+type WorkspaceConnectionLatencyMS struct {
+	P50 float64
+	P95 float64
+}
+
+type DeploymentStats struct {
+	// AggregatedFrom is the time in which stats are aggregated from.
+	// This might be back in time a specific duration or interval.
+	AggregatedFrom time.Time `json:"aggregated_since" format:"date-time"`
```

Comment:
Any reason to keep json/field out of sync?

Reply:
Nope, just a mistake on my end. Good catch!